vendor/github.com/xeipuuv/gojsonschema/json_schema_test_suite/LICENSE

```diff
@@ -1,19 +0,0 @@
-Copyright (c) 2012 Julian Berman
```

Comment:
This definitely looks like a bug in go modules (also as outlined above); this file has a standard name, so should not be removed by `go mod vendor`; I'll create a minimal reproducer and open a bug-report.

Reply:
Ah, actually; ignore this: looks like the associated files are not vendored, so it makes sense to not vendor the LICENSE as well